rest_framework/fields.py

```diff
@@ -639,20 +639,37 @@ def __init__(self, **kwargs):
 
 
 class UUIDField(Field):
+    valid_formats = ('hex_verbose', 'hex', 'int', 'urn')
+
     default_error_messages = {
         'invalid': _('"{value}" is not a valid UUID.'),
     }
 
+    def __init__(self, **kwargs):
+        self.uuid_format = kwargs.pop('format', None) or 'hex_verbose'
```

Comment:
No, because explicitly passing `format=None` in that case will result in an incorrect value for `format`.